bugbug/tools/code_review.py

```diff
@@ -1322,3 +1323,53 @@ def find_similar_patch_comments(self, patch: Patch, limit: int):
                         max_score_per_comment[result.id] = result
 
         return sorted(max_score_per_comment.values())[-limit:]
+
+
+class EvaluationAction(enum.Enum):
+    APPROVE = 1
+    REJECT = 2
+    IGNORE = 3
+
+
+@dataclass
+class SuggestionFeedback:
+    id: int
+    comment: str
+    file_path: str
+    action: Literal["APPROVE", "REJECT", "IGNORE"]
+    user: str
+
+
+class SuggestionsFeedbackDB:
+    def __init__(self, vector_db: VectorDB) -> None:
+        self.vector_db = vector_db
+        self.embeddings = OpenAIEmbeddings(
+            model="text-embedding-3-large", api_key=get_secret("OPENAI_API_KEY")
```

Comment:
We should make this configurable, or at least define it in a constant

Reply:
We could use dependency injection for that. We could pass the embedding class to the contractor, allowing more flexibility in selecting the provider, not just the model.